### PR TITLE
Feat : Input, Images and headings added

### PR DESCRIPTION
**Pull request by yashodaAgarwal**
Please review my code.

### CODE REVIEW

File: style/Badges.css [hunk 1]
```diff
@@ -4,12 +4,11 @@
   padding-top: 20px;
   padding-bottom: 20px;
   background-color: transparent;
-  border: 0.5px solid rgba(128, 128, 128, 0.363);
+  border: 0.5px solid var(--border-color);
   margin: 10px;
 }
```
Comment (m26unkwn) [inline style/Badges.css]: Here for padding property you can use 
padding: 20px 0;  DRY

File: style/Alerts.css [hunk 2]
```diff
@@ -23,7 +23,7 @@
     padding:10px 10px 10px 10px;
     margin:20px 0px 20px 0px;
```
Comment (m26unkwn) [inline style/Alerts.css:24]: You can also rem units

**Review by m26unkwn — APPROVED** [review]
Otherwise everything is GTM.

File: style/Badges.css (hunk 1 above)
Comment (yashodaAgarwal) [reply style/Badges.css]: done thankyou

File: style/Alerts.css (hunk 2 above)
Comment (yashodaAgarwal) [reply style/Alerts.css:24]: Okay thankyou for suggestion

File: style/Typography.css [hunk 3]
```diff
@@ -0,0 +1,42 @@
+.Typo-items{
+    padding:20px;
+    border: 0.5px solid var(--border-color);
+    margin: 20px;
+    display:flex;
+    flex-direction: column;
+    justify-content: space-between;
+    height: 350px;
+    font-weight: 600px;
```
Comment (nisargpawade) [inline style/Typography.css]: this won't work, remove px from font weight

File: style/Input.css [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+width: 20%;
+}
+.Error-message{
+    color:var(--red-color);
+    font-size: 15px;
+}
+.validation-msg{
+    color:var(--yellow-color);
+    font-size: 15px;
+}
```
Comment (nisargpawade) [inline style/Input.css]: either increase their font weight or use darker shade of yellow for better visibility

**Review by nisargpawade — APPROVED** [review]

File: style/Typography.css (hunk 3 above)
Comment (yashodaAgarwal) [reply style/Typography.css]: oh thanks that was by mistake

File: style/Input.css (hunk 4 above)
Comment (yashodaAgarwal) [reply style/Input.css]: okay

**Pull request merged**